### PR TITLE
ptex: fix test for Linux

### PR DESCRIPTION
**Pull request by alebcay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by carlocab — DISMISSED** [review]

**Comment by carlocab** [comment]
Actually, while we're here, can we change the Git clone to a tarball? It doesn't look like there are any submodules upstream.

File: Formula/ptex.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
 class Ptex < Formula
   desc "Texture mapping system"
   homepage "https://ptex.us/"
-  url "https://github.com/wdas/ptex.git",
-      tag:      "v2.4.1",
-      revision: "93c8bad39a6122c42c9d9d8e29d715bd73a6c575"
-  license "BSD-3-Clause"
+  url "https://github.com/wdas/ptex/archive/refs/tags/v2.4.1.tar.gz"
+  sha256 "664253b84121251fee2961977fe7cf336b71cd846dc235cd0f4e54a0c566084e"
```
Comment (carlocab) [inline Formula/ptex.rb:5]:
```suggestion
  sha256 "664253b84121251fee2961977fe7cf336b71cd846dc235cd0f4e54a0c566084e"
  license "BSD-3-Clause"
```
I think you dropped this

Comment (alebcay) [reply Formula/ptex.rb:5]: Indeed I did, surprised the CI didn't yell.

**Review by SMillerDev — APPROVED** [review]

**Comment by alebcay** [comment]
https://github.com/Homebrew/homebrew-core/actions/runs/1718282381

**Pull request closed without merging**